src/plot/axisannotation.js

```diff
@@ -138,31 +138,19 @@ function backgroundPath(accessor, axis, height, width, point, neg) {
         if(height/2 < point) pt = height/2;
         else h = height/2-point;
 
-        return [
-          'M', 0, value,
-          'l', neg*Math.max(axis.innerTickSize(), 1), -pt,
-          'l', 0, -h,
-          'l', neg*width, 0,
-          'l', 0, height,
-          'l', neg*-width, 0,
-          'l', 0, -h
-        ].join(' ');
+        return 'M 0 ' + value + ' l ' + String(neg*Math.max(axis.innerTickSize(), 1)) + ' ' + String(-pt) +
```

Comment:
Test: https://gist.github.com/andredumas/7a3277e3cb06c35f4ba34562a873cf35#file-number-tostring-html-L25

Pls flick through the tests make sure I'm not doing anything crazy for these suggestions. These area's are key to responsiveness feel for the chart. This is a really good step forward.